update overview and add diagram (#37)
ministryofjustice/find-moj-data-user-guide: source/documentation/index.md

# Find MoJ data 

Find MoJ data is a searchable catalogue service to help users find and understand data, supporting the [Ministry of Justice's 2025 data strategy](https://www.gov.uk/government/publications/ministry-of-justice-digital-strategy-2025). 
You can [read an update on the MoJ’s data strategy](https://mojdigital.blog.gov.uk/2023/12/19/mojs-data-strategy-one-year-on/) to learn more about Find MoJ data’s role in improving the department’s approach to data. 

Find MoJ data unites many different MoJ sources in one place, describing datasets with metadata but without including actual data. 
This is because data owners are responsible for controlling access to potentially sensitive data.  

You can use Find MoJ data to:

* discover datasets from [MoJ data sources](#moj-data-sources)
* evaluate datasets for relevance without needing access to the data they contain 
* locate data owners, so you can request access or report a problem

##  Using Find MoJ data for your work

> Anyone with an @justice.gov.uk (Microsoft) account can sign in to Find MoJ data. Use the service’s search bar to discover datasets through keywords, or select a category to see all datasets in one area. You can filter results to focus on relevant datasets.  

Reviewer: ```suggestion
Anyone with an @justice.gov.uk (Microsoft) account has access to Find MoJ data. Use the service’s search bar to discover datasets through keywords, or select a category to see all datasets in one area. You can filter results to focus on relevant datasets.  
```

> Each entry lists metadata to describe the data assets, so you can understand a dataset’s scope and the type of information it contains. For example, Find MoJ data can show you the names of tables and columns in a dataset as well as the type of input, such as `string` or `integer`. 

Reviewer: ```suggestion
Each entry lists metadata to describe the data asset, so you can understand an asset's scope and the type of information it contains. For example, Find MoJ data can show you database descriptions for versions of NOMIS. It can also show you table descriptions and schema information including datatypes for tables within NOMIS.
```

You can also see other information about a dataset, including:

> * format type	

Reviewer: ```suggestion
* format type	
* table descriptions
* field descriptions
```
While the completion rate of descriptions is variable, it's often what people want

* table schemas	
* security classification 
* when Find MoJ data last checked for an update	
* how regularly the data owner intends to update the data

## How Find MoJ data works

<img src="/images/how-find-moj-data-works.svg" alt="How Find MoJ data works" width="400" height="250">

1. Data owners store and maintain their datasets on the Analytical Platform (AP) and other sources.

> 1. Find MoJ data ingests the metadata of datasets from the AP and other sources. 

Reviewer: ```suggestion
1. Find MoJ data ingests the metadata of datasets from the AP and other sources. Ingestions are run daily.
```

1. Find MoJ data adds the ingested metadata to DataHub so it can be accessed from the DataHub API. 
1. Information about ingested datasets is displayed in Find MoJ data’s user interface.
1. Users can find and evaluate dataset entries on Find MoJ data. 
1. Users can contact data owners to request access or report a problem. 

## MoJ data sources

> The MoJ contains a large number of potential data sources following different approaches, so the data landscape is complex and difficult to join up. While the service is in private beta, the catalogue includes a limited range of datasets as testing continues with users. 

Reviewer: ```suggestion
The MoJ owns a large number of potential data sources which are stored and maintained differently, which makes the data landscape complex and difficult to join up. While the service is in private beta, the catalogue includes a limited range of datasets geared towards AP users as testing continues. 
```
Wording sounded funny to me. If you don't like the suggestion feel free to ignore this one.

Find MoJ data includes information about the following:

* statistical publications from GOV.UK
* charts from [Justice Data](https://data.justice.gov.uk/) 
* digital prisons reporting (DPR) data from the [Create a Derived Table (CaDeT) service](https://github.com/moj-analytical-services/create-a-derived-table)
* electronic monitoring (EM) data from the [Create a Derived Table (CaDeT) service](https://github.com/moj-analytical-services/create-a-derived-table)
* models from the [Create a Derived Table (CaDeT) service](https://github.com/moj-analytical-services/create-a-derived-table)
* AWS Glue databases, including from legacy pipelines

> * selected Postgres databases from HMPPS

Reviewer: ```suggestion
Coming soong:
* selected Postgres databases from HMPPS
```

Author: Discussed and agreed not to add now as not live yet. I've created a subtask on main issue for Postgres databases to update the docs when it does go live.

## Adding datasets to Find MoJ data's catalogue

You can add your data to Find MoJ data's catalogue yourself if it's stored on the Analytical Platform's Create a Derived Table (CaDeT) service. Follow guidance on [Adding metadata from Create a Derived Table to Find MoJ data](./cadet-registration.md)

> If your data is somewhere else, [contact us](#contact-us) to discuss. 

Reviewer: ```suggestion
If your data is somewhere else eg Glue, RDS, or an API, [contact us](#contact-us) to discuss. 
```

1. Information about ingested datasets is displayed in Find MoJ data’s user interface.
1. Users can find and evaluate dataset entries on Find MoJ data. 
1. Users can contact data owners to request access or report a problem. 

## MoJ data sources

The MoJ owns a large number of data sources which are stored and maintained differently, making the data landscape complex and difficult to join up. While the service is in private beta, the catalogue includes a limited range of datasets. 

Find MoJ data includes information about the following:

* statistical publications from GOV.UK
* charts from [Justice Data](https://data.justice.gov.uk/) 
* digital prisons reporting (DPR) data from the [Create a Derived Table (CaDeT) service](https://github.com/moj-analytical-services/create-a-derived-table)
* electronic monitoring (EM) data from the [Create a Derived Table (CaDeT) service](https://github.com/moj-analytical-services/create-a-derived-table)
* models from the [Create a Derived Table (CaDeT) service](https://github.com/moj-analytical-services/create-a-derived-table)
* AWS Glue databases, including from legacy pipelines

## Adding datasets to Find MoJ data's catalogue

> You can add an entry about your data to Find MoJ data's catalogue yourself if it's stored on the Analytical Platform's Create a Derived Table (CaDeT) service. Follow guidance on [Adding metadata from Create a Derived Table to Find MoJ data](./cadet-registration.md).

Reviewer: ```suggestion
You can add an entry about your data to Find MoJ data's catalogue yourself if it's derived using the Analytical Platform's Create a Derived Table (CaDeT) service. Follow guidance on [Adding metadata from Create a Derived Table to Find MoJ data](./cadet-registration.md).
```
